bin/crosses_and_noughts.dart

```diff
@@ -52,52 +70,55 @@ void drawField(List<List<Square>> field) {
   }
 }
 
+//Метод isWon проверяет условия победы
 bool isWon(List<List<Square>> field) {
   if ((field[0][0].content == field[0][1].content) &&
       (field[0][0].content == field[0][2].content) &&
       (field[0][0].content != Figures.empty)) {
     return true;
   }
-  ;
+
 
   if ((field[0][0].content == field[1][0].content) &&
       (field[0][0].content == field[2][0].content) &&
       (field[0][0].content != Figures.empty)) {
     return true;
   }
-  ;
+ 
 
   if ((field[1][0].content == field[1][1].content) &&
       (field[1][0].content == field[1][2].content) &&
       (field[1][0].content != Figures.empty)) {
     return true;
   }
-  ;
+ 
 
   if ((field[2][0].content == field[2][1].content) &&
       (field[2][0].content == field[2][2].content) &&
       (field[2][0].content != Figures.empty)) {
     return true;
   }
-  ;
+  
 
   if ((field[0][0].content == field[1][1].content) &&
       (field[0][0].content == field[2][2].content) &&
       (field[0][0].content != Figures.empty)) {
     return true;
   }
-  ;
+  
 
   if ((field[2][0].content == field[1][1].content) &&
       (field[2][0].content == field[0][2].content) &&
       (field[2][0].content != Figures.empty)) {
     return true;
   }
-  ;
+  
 
   return false;
 }
 
+
+//Метод isDraw проверяет условия ничьи
 bool isDraw(List<List<Square>> field) {
```

Comment:
Ты уверен что true у тебя выйдет по умолчанию?
```suggestion
bool isDraw(List<List<Square>> field) {
    return !field.contains(Figures.empty);
```